### PR TITLE
certinspection: use functions from library-go

### PR DESCRIPTION
**Pull request by vrutkovs**
Follow up for https://github.com/openshift/library-go/pull/1568

### CODE REVIEW

**Comment by vrutkovs** [comment]
/hold

still testing this

File: pkg/cmd/certinspection/certinspection.go [hunk 1]
```diff
@@ -106,8 +108,8 @@ func (o *CertInspectionOptions) Run() error {
 			inspectConfigMap(castObj)
 		case *corev1.Secret:
 			inspectSecret(castObj)
-		case *certificatesv1beta1.CertificateSigningRequest:
-			inspectCSR(castObj)
+		// case *certificatesv1beta1.CertificateSigningRequest:
```
Comment (deads2k) [inline pkg/cmd/certinspection/certinspection.go:111]: huh, I forgot to move this?

**Comment by deads2k** [comment]
/lgtm

I'm willing to accept the changes here when they compile and you're ready.

File: pkg/cmd/certinspection/certinspection.go (hunk 1 above)
Comment (vrutkovs) [reply pkg/cmd/certinspection/certinspection.go:111]: re-adding this in https://github.com/openshift/library-go/pull/1581 (also with v1beta1 and v1 support)

**Pull request merged**